src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: string[];
+  perPage: number;
+  currentPage: number;
+  onPageChange: (n: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total.length / perPage);
+  const pagesToRender = getNumbers(1, numberOfPages);
+
+  const sliceFrom = currentPage * perPage - perPage;
+  const sliceTo = currentPage * perPage;
+  const slicedItems = total.slice(sliceFrom, sliceTo);
+
+  const handleBack = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleForward = () => {
+    if (currentPage < numberOfPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', { disabled: currentPage === 1 })}
+          onClick={handleBack}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
```

Comment:
`currentPage === 1` you usse twice, create a variable, and the same for the last page